clang/lib/CodeGen/CGHLSLRuntime.cpp

```diff
@@ -381,6 +381,20 @@ void CGHLSLRuntime::addBuffer(const HLSLBufferDecl *BufDecl) {
   }
 }
 
+void CGHLSLRuntime::addRootSignature(
+    const HLSLRootSignatureDecl *SignatureDecl) {
+  llvm::Module &M = CGM.getModule();
+  Triple T(M.getTargetTriple());
+
+  // If we are not targeting a root signature enviornment then this decl will
+  // be generated when the function decl it is attached is handled
```

Comment:
Comment feels a little vebose would this be the same as saying
```suggestion
  // Generated later with the function decl if not targeting root signature
```